types/bun/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "private": true,
+    "name": "@types/bun",
+    "version": "1.0.9999",
+    "projects": [
+        "https://bun.sh"
+    ],
+    "dependencies": {
+        "@types/node": "*",
+        "@types/ws": "*",
+        "undici-types": "~5.26.4"
+    },
+    "devDependencies": {
+        "@types/bun": "workspace:."
+    },
+    "owners": [
```

Comment:
all those users would receive notification, is that ok for them?
Could we one get their ack here? (or should you just give them credits)
https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Reply:
Yes, those are all Bun team members. We want to be able to merge definition PRs quickly so some redundancy here is fine.